<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>460</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
  background-color: #0095ff;
 border-radius: 12px;
  border-style: none;
  box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  display: inline-block;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,&quot;Liberation Sans&quot;,sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 1.15385;
  margin: 0;
  outline: none;
  padding: 8px .8em;
  position: relative;
  text-align: center;
  text-decoration: none;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  vertical-align: baseline;
  white-space: nowrap;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #07c;
}

QPushButton:focus {
  box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
}

QPushButton:pressed {
  background-color: #0064bd;
  box-shadow: none;
}</string>
    </property>
    <property name="text">
     <string>Vote</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createIdeasPushButtob">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton {
  background-color: #0095ff;
 border-radius: 12px;
  border-style: none;
  box-shadow: rgba(255, 255, 255, .4) 0 1px 0 0 inset;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  display: inline-block;
  font-family: -apple-system,system-ui,&quot;Segoe UI&quot;,&quot;Liberation Sans&quot;,sans-serif;
  font-size: 20px;
  font-weight: 500;
  line-height: 1.15385;
  margin: 0;
  outline: none;
  padding: 8px .8em;
  position: relative;
  text-align: center;
  text-decoration: none;
  user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  vertical-align: baseline;
  white-space: nowrap;
}

QPushButton:hover,
QPushButton:focus {
  background-color: #07c;
}

QPushButton:focus {
  box-shadow: 0 0 0 4px rgba(0, 149, 255, .15);
}

QPushButton:pressed {
  background-color: #0064bd;
  box-shadow: none;
}</string>
    </property>
    <property name="text">
     <string>Create Ideas</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="bestIdeaLineEdit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>400</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>341</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
    <property name="text">
     <string>Ideas:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>201</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
    <property name="text">
     <string>For Voting
Enter Number of
Best Ideas</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="voatingResultPlainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>341</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
  border: 4px solid blue;;
  border-radius: 12px;
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>359</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
